model/policy.go

```diff
@@ -179,10 +179,15 @@ func (model Model) RemoveFilteredPolicy(sec string, ptype string, fieldIndex int
 	var effects [][]string
 	res := false
 	firstIndex := -1
+
+	if len(fieldValues) == 0 {
+		return false, effects
+	}
+
 	for index, rule := range model[sec][ptype].Policy {
 		matched := true
 		for i, fieldValue := range fieldValues {
-			if fieldValue != "" && rule[fieldIndex+i] != fieldValue {
+			if rule[fieldIndex+i] != fieldValue {
```

Comment:
Why remove it? I think the empty string "" here means a placeholder that we don't want to do comparison for this column.

Reply:
we should add check for `len(fieldValues)`

Reply:
fix

Reply:
> we should add check for `len(fieldValues)`

I added `len(fieldValues)`.